docs/source/faq.md

```diff
@@ -275,7 +275,8 @@ Yes.
 When adaptive routing is configured on an Infiniband fabric, it is enabled per SL 
 (IB Service Layer).  
 Setting `UCX_IB_SL=<sl-num>` will make UCX run on the given
-service level and utilize adaptive routing. 
+service level and utilize adaptive routing. On RoCE, setting `UCX_IB_AR_ENABLE`
+can activate adaptive routing without forcing any SL.
```

Comment:
Need to update the whole paragraph, can use chatgpt to rephrase:
UCX_IB_AR_ENABLE enables adataptive routing on both Infiniband and RoCE clusters. On Infiniband, it will try to select the first SL that has adaptive routing enabled, and on RoCE it will enable adaptive routing if supported by HW configuration. if UCX_IB_AR_ENABLE is set to yes, and adaptive routing is not supported by the underlying network, it will cause an error. When set to try, lack of support will be silently ignored.

Reply:
fixed